tests/test_iri.py

```diff
@@ -19,9 +19,11 @@
         (u'http://βόλος.com/β/ό?λ#ος', u'http://xn--nxasmm1c.com/%CE%B2/%CF%8C?%CE%BB#%CE%BF%CF%82'),
         (u'http://ශ්\u200dරී.com', u'http://xn--10cl1a0b660p.com'),
         (u'http://نامه\u200cای.com', u'http://xn--mgba3gch31f060k.com'),
-        (u'http://Bü:ẞ@gOoGle.com', u'http://B%C3%BC:%E1%BA%9E@google.com'),
+        (u'http://Bü:ẞ@gOoGle.com', u'http://B%C3%BC:%E1%BA%9E@gOoGle.com'),
```

Comment:
So, in the real world, `gOoGle.com` and `google.com` are equivalent. I'd expect that calling `.normalize()` would, however, lower-case these if `.encode()` doesn't auto-normalize.

Also, does `http+unix` actually encode the `/` to turn the file path into a hostname? I don't think I've used that protocol enough to know one way or the other.

Reply:
Right, I figure the `.encode()` shouldn't be doing normalization that it doesn't *have* to do. Because urllib3 only normalizes http or https we should be fine here.

That's a good question, I would assume no because if it did I don't know how you'd distinguish the "hostname" from the path from an http+unix URI.